arelle/CntlrCmdLine.py

```diff
@@ -317,11 +319,9 @@ def parseArgs(args):
                              "or ../examples/plugin/hello_dolly.py for relative use of examples directory) "
                              "Local python files do not require .py suffix, e.g., hello_dolly without .py is sufficient, "
                              "Packaged plug-in urls are their directory's url (e.g., --plugins EdgarRenderer or --plugins xbrlDB).  " ))
-    parser.add_option("--packages", "--package", action="store", dest="packages",
-                      help=_("Specify taxonomy packages configuration.  "
-                             "Enter 'show' to show current packages configuration.  "
-                             "Commands show, and module urls are '|' separated: "
-                             "url specifies a package by its url or filename, please use full paths. "
+    parser.add_option("--packages", "--package", action="append", dest="packages",
+                      help=_("Load taxonomy packages. Option can be repeated for multiple files. "
```

Comment:
Can you also update the documentation for this? It's unfortunate that it's duplicated. https://github.com/Arelle/Arelle/blob/3df8202a6fc1843001648e44714c1bb72c75aac2/docs/source/command_line.md?plain=1#L147

Reply:
Ooh, nice, I didn't realise we now have proper docs for the command line.